Dockerfile

```diff
@@ -12,22 +12,22 @@ LABEL org.label-schema.maintainer="Voxpupuli Team <info@voxpupuli.org>" \
       org.label-schema.dockerfile="/Dockerfile"
 
 ARG RUBYGEM_PUPPET
-ENV RUBYGEM_PUPPET=${RUBYGEM_PUPPET:-8.6.0}
+ENV RUBYGEM_PUPPET=${RUBYGEM_PUPPET:-8.8.1}
 
 ARG RUBYGEM_FACTER
-ENV RUBYGEM_FACTER=${RUBYGEM_FACTER:-4.7.0}
+ENV RUBYGEM_FACTER=${RUBYGEM_FACTER:-4.8.0}
 
 ARG RUBYGEM_VOXPUPULI_TEST
-ENV RUBYGEM_VOXPUPULI_TEST=${RUBYGEM_VOXPUPULI_TEST:-7.1.0}
+ENV RUBYGEM_VOXPUPULI_TEST=${RUBYGEM_VOXPUPULI_TEST:-9.0.0}
```

Comment:
This is a really big change that drops legacy facts support for unit testing.

Reply:
that might be worth a major release

Reply:
will do a major